### ISSUE TITLE
Filter activity via api

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#1733]
https://github.com/nextcloud/activity/pull/174

Then we can filter the activities via api, eg. favorites, file changes, ...

UI should look like, as discussed with @jancborchardt:
![image](https://user-images.githubusercontent.com/5836855/32277394-9dc4da26-bf12-11e7-9838-372d7896431e.png)

**Comment by AndyScherzinger** [comment]
Looks good to me, some minor questions though:

* Should a card layout be used?
* Should the card layout be used "per application"
* Should the headline be capitalized (had this discussion with @jancborchardt and others for settings and we agreed on not capitalized)
* Order by / group by app or time/date?
* Should the time be on top (like in the screenshot) or at the bottom of the item (or bottom right) like in other views that display items including a timestamp?
* Is this for activities or notifications or both (since it says Notifications in the action bar but the referenced issue is about activities)
* How is read, unread and participating to be distinguished visually?
* is the headline icon to be circular via the client or is this just displayed the way we get it from the server?

**Comment by tobiasKaminsky** [comment]
@jancborchardt for the most of it ;-)

But to clarify my first post: the distinction/switch between "favorites, file changes", ... should be done like switcher in the screenshot. The other UI can (and should) be done like in our current app.

> How is read, unread and participating to be distinguished visually?

Where is this in activity stream?

**Comment by jancborchardt** [comment]
Yeah, disregard the content view in the above screenshot - we keep the main content as it is. :) It's just about the Radiobutton menu.

**Comment by AndyScherzinger** [comment]
Ah okay 👍
